packages/webview/src/styles/subflow-config-popup.css

```
@@ -320,3 +321,212 @@
 .subflow-config-popup .monaco-scrollable-element > div[class*="scrollbar"][role="presentation"] > div[class*="slider"]:hover {
   background: #94a3b8 !important;
 }
+
+/* ===== Dark Theme ===== */
+body[data-vscode-theme-kind="vscode-dark"] .subflow-config-popup,
+body[data-vscode-theme-kind="vscode-high-contrast"] .subflow-config-popup {
+  background: rgba(30, 30, 46, 0.95);
```

Stylelint (17.2.0)

[error] Expected "0.95" to be "95%" (alpha-value-notation)

(alpha-value-notation)